wave_benchmark/include/wave/benchmark/pose_measurement.hpp

```diff
@@ -0,0 +1,49 @@
+#ifndef WAVE_BENCHMARK_POSE_MEASUREMENT_HPP
+#define WAVE_BENCHMARK_POSE_MEASUREMENT_HPP
+
+#include <chrono>
+#include "wave/geometry/rotation.hpp"
+#include "wave/utils/math.hpp"
+
+namespace wave {
+
+using TimeType = std::chrono::steady_clock::time_point;
+
+struct Pose {
+    wave::Rotation rotation;
+    wave::Vec3 translation;
+};
+
+struct PoseMeasurement {
```

Comment:
Not really. The ground truth is probably coming from GPS but it doesn't have to. They are just generic poses.